src/main/java/hudson/tasks/test/TestResultProjectAction.java

```diff
@@ -102,6 +102,12 @@ public AbstractTestResultAction getLastTestResultAction() {
         return null;
     }
 
+    @SuppressWarnings("unchecked")
+    public Class<T> getTestResultActionClass() {
```

Comment:
Would be nice to add some documentation for this method. I'm pretty sure all classes in other plugins are supposed to override this method in order to have the correct behavior